storybook/src/components/pagination/pagination.css-template.ts

```diff
@@ -8,65 +8,70 @@ export const cssPagination: ComponentImplementation<Pagination> = {
   template: () =>
     function paginationTemplate({ currentPage, totalPages }) {
       return html`
-        <ul class="pagination">
-          ${currentPage !== 1
-            ? html`
-                <li>
-                  <a href="#" aria-label="Vorige">
-                    <span class="dso-previous" aria-hidden="true"></span>
-                  </a>
-                </li>
-                <li>
-                  <a href="#">1</a>
-                </li>
-              `
-            : nothing}
-          ${(currentPage !== totalPages || totalPages >= 4) && currentPage === totalPages && currentPage !== 1
-            ? html`
-                <li>
-                  <a href="#">${currentPage - 2}</a>
-                </li>
-              `
-            : nothing}
-          ${currentPage >= 3 &&
-          ((currentPage >= 3 && totalPages <= 5) || (totalPages > 5 && currentPage !== totalPages - 2))
-            ? html`
-                <li>
-                  <a href="#">${currentPage - 1}</a>
-                </li>
-              `
-            : nothing}
-          <li class="active">
-            <span aria-current="page">${currentPage}</span>
-          </li>
-          ${currentPage <= totalPages - 2 &&
-          ((currentPage <= 3 && totalPages <= 5) || (currentPage !== 3 && totalPages > 5))
-            ? html`
-                <li>
-                  <a href="#">${currentPage + 1}</a>
-                </li>
-              `
-            : nothing}
-          ${currentPage === 1 && totalPages > currentPage + 2
-            ? html`
-                <li>
-                  <a href="#">${currentPage + 2}</a>
-                </li>
-              `
-            : nothing}
-          ${currentPage !== totalPages
-            ? html`
-                <li>
-                  <a href="#">${totalPages}</a>
-                </li>
-                <li>
-                  <a href="#" aria-label="Volgende">
-                    <span class="dso-next" aria-hidden="true"></span>
-                  </a>
-                </li>
-              `
-            : nothing}
-        </ul>
+        <nav aria-label="Paginering">
+          <ul class="pagination">
```

Comment:
Alle implementaties hebben een selector die matcht op het root element.

Met deze change willen we aangeven dat elke Pagination vanaf nu begint met een `<nav>`. Echter, de root selector die dat element aanwijst is niet verschoven.

Ik had dus eigenlijk de volgende opbouw verwacht:

```suggestion
        <nav aria-label="Paginering" class="pagination">
          <ul>
```

En dat de styling selectors natuurlijk hierin meegingen. Hoe denk jij hierover?

_(idealiter grijp ik deze kans aan om de root selector te renamen naar `dso-pagination`, maar dat terzijde)_

Reply:
Dan moet de styling behoorlijk omgegooid worden, want die is gebouwd om een `<ul>` te stylen.

Reply:
valt mee. done :)